src/core/Akka.Tests/IO/UdpConnectedIntegrationSpec.cs

```diff
@@ -158,43 +171,36 @@ public void The_UDP_connection_oriented_implementation_must_not_leak_memory()
             const int batchCount = 2000;
             const int batchSize = 100;
             
-            var serverAddress = _addresses[0];
-            var clientAddress = _addresses[1];
             var udpConnection = UdpConnected.Instance.Apply(Sys);
-            
             var poolInfo = udpConnection.SocketEventArgsPool.BufferPoolInfo;
             poolInfo.Type.Should().Be(typeof(DirectBufferPool));
             poolInfo.Free.Should().Be(poolInfo.TotalSize);
             poolInfo.Used.Should().Be(0);
             
-            var server = CreateTestProbe();
-            udpConnection.Manager.Tell(new UdpConnected.Connect(server, clientAddress, serverAddress), server);
-            server.ExpectMsg<UdpConnected.Connected>();
-            var serverEp = server.LastSender;
-
-            var client = CreateTestProbe();
-            udpConnection.Manager.Tell(new UdpConnected.Connect(client, serverAddress, clientAddress), client);
-            client.ExpectMsg<UdpConnected.Connected>();
-            var clientEp = client.LastSender;
+            var serverProbe = CreateTestProbe();
+            var (server, serverEndPoint) = BindUdp(serverProbe);
+
+            var clientProbe = CreateTestProbe();
+            var (client, clientEndPoint) = ConnectUdp(serverEndPoint, clientProbe);
```

Comment:
Changed this from using UdpConnected both ways to just one way because using connected both ways would require us to know both ports ahead of time, which is racy.